src/Turbo/src/DependencyInjection/Compiler/RegisterMercureHubs.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\UX\Turbo\Bridge\Mercure\Broadcaster;
+use Symfony\UX\Turbo\Bridge\Mercure\TurboStreamListenRenderer;
+
+/**
+ * This compiler pass ensures that TurboStreamListenRenderer
+ * and Broadcast are registered per Mercure hub.
+ *
+ * @author Pierre Ambroise<pierre27.ambroise@gmail.com>
+ */
+final class RegisterMercureHubs implements CompilerPassInterface
```

Comment:
```suggestion
final class RegisterMercureHubsPass implements CompilerPassInterface
```

I know we have a couple without in the UX repository right now, but it feels more coherent to follow Symfony codebase conventions